instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientMetrics.java

```diff
@@ -67,8 +67,13 @@ public Context start(Context context, Attributes startAttributes, long startNano
   public void end(Context context, Attributes endAttributes, long endNanos) {
     State state = context.get(HTTP_CLIENT_REQUEST_METRICS_STATE);
     if (state == null) {
-      logger.debug(
-          "No state present when ending context {}. Cannot record HTTP request metrics.", context);
+      if (logger.isLoggable(Level.FINE)) {
+        logger.log(
+            Level.FINE,
+            "No state present when ending context "
+                + context
+                + ". Cannot record HTTP request metrics.");
+      }
```

Comment:
I think can parameterize using `{0}` and avoid many of the conditionals

Reply:
AFAIR slf4j (which we're _actually_ using underneath) only recognizes `{}` placeholders - it will treat `{0}` (`{anything}`) literally.

Reply:
Oh, looks like `PatchLogger` already handles that -- my bad, I'll change that PR to use placeholders.